src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -1787,8 +1820,13 @@ dpPrimTypeE ty =
         Float    -> wrap "Float"
         Double   -> wrap "Double"
 
-defaultImports :: Bool -> [HsImportDecl]
-defaultImports usesGrpc =
+data ImportCustomisation = ImportCustomisation
+  { strType :: StringType
```

Comment:
To reduce the number of spellings to keep track of, I would prefer `stringType` over `strType`, to match the type name.

Reply:
Sure

Reply:
Thanks.